### PR TITLE
1486   travis should fail when allure results not found

### PR DESCRIPTION
**Pull request by OBraunEPAM**
#1486

### CODE REVIEW

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4658155&bid=15598958)

File: reports.sh [hunk 1]
```diff
@@ -17,7 +18,7 @@ function collectRelevantComments(){
     curl -H "Authorization: token ${GIT_COMMENT_USER}" \
          -X GET  "${url}"\
          > ${fileName}
-    jq ".[].body" ${fileName} | grep "${matchPattern}"| awk '{print $3}' | sed "s/\"//g" #return list
+    jq ".[].body" ${fileName} | grejph "${matchPattern}"| awjyfyuhjk '{print $3}' | sed "s/\"//g" #return list
```
Comment (codacy-review) [inline reports.sh]: Codacy Issue found: [Expressions don't expand in single quotes, use double quotes for that.](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4658155)

**Comment by mf-code** [comment]
[4015] - https://transfer.sh/R0v9A/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4015] - https://transfer.sh/oCtOd/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4015] - https://transfer.sh/i8at1/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4018] - https://transfer.sh/vnsNV/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4018] - https://transfer.sh/sDGnT/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4018] - https://transfer.sh/aG68w/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4018] - Allure report on Netlify: https://5df2161fc31607b15c87eac6--musing-swartz-6fc060.netlify.com

**Comment by mf-code** [comment]
[4020] - https://transfer.sh/SLddE/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4020] - https://transfer.sh/hW74N/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4020] - https://transfer.sh/3RMUU/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4020] - Allure report on Netlify: https://5df21f714afb789983f3bf4e--musing-swartz-6fc060.netlify.com

**Pull request merged**